File: node/7 raphael/public/js/microgear.js
  const APPID     = "seniorProject";
  const APPKEY    = "iAeBgu6Vuvl7TKs";
  const APPSECRET = "IfH0R3EnBBUJC27NTIr632TOm";

    var microgear = Microgear.create({
        key: APPKEY,
        secret: APPSECRET,
        alias : "htmlgear"         /*  optional  */
    });

    microgear.on('message',function(topic,msg) {
        console.log(msg);
        showpoint(msg);
    });

    microgear.on('connected', function() {
        microgear.setAlias('htmlgear');
    });

    microgear.on('present', function(event) {
        console.log(event);
    });

    microgear.on('absent', function(event) {
        console.log(event);
    });

    microgear.connect(APPID);
    microgear.subscribe("/web");

File: node/microgear.py
import microgear.client as microgear

appid = "seniorProject"
gearkey = "XqUHLlu0jVLlfzY"
gearsecret =  "wDnkuU5WAkashQek9wXx4NIU9"

microgear.create(gearkey,gearsecret,appid,{'debugmode': True})

def connection():
  print "Now I am connected with netpie"

def subscription(topic,message):
  print topic+" "+message

def disconnect():
  print "disconnect is work"

microgear.setalias("test_sent")
microgear.on_connect = connection
microgear.on_message = subscription
microgear.on_disconnect = disconnect
microgear.subscribe("/web")
microgear.connect(True)